src/libxtp/threecenter_gwbse.cc

```diff
@@ -99,13 +152,11 @@ void TCMatrix_gwbse::Fill(const AOBasis& gwbasis, const AOBasis& dftbasis,
 /*
  * Determines the 3-center integrals for a given shell in the GW basis
  * by calculating the 3-center overlap integral of the functions in the
- * GW shell with ALL functions in the DFT basis set (FillThreeCenterOLBlock),
- * followed by a convolution of those with the DFT orbital coefficients
+ * GW shell with ALL functions in the DFT basis set (FillThreeCenterOLBlock)
  */
-
-void TCMatrix_gwbse::FillBlock(std::vector<Eigen::MatrixXd>& block,
-                               const AOShell& auxshell, const AOBasis& dftbasis,
-                               const Eigen::MatrixXd& dft_orbitals) {
+std::vector<Eigen::MatrixXd> TCMatrix_gwbse::ComputeSymmStorage(
+    const AOShell& auxshell, const AOBasis& dftbasis,
+    const Eigen::MatrixXd& dft_orbitals) const {
   std::vector<Eigen::MatrixXd> symmstorage;
```

Comment:
Can you replace this ` push_back` by ` std::vector<Eigen::MatrixXd> symmstorage=std::vector<Eigen::MatrixXd>(auxshell.getNumFunc(),Eigen::MatrixXd::Zero(dftbasis.AOBasisSize(), dftbasis.AOBasisSize()))`

Reply:
thanks!